packages/blade/src/components/Badge/Badge.tsx

```diff
@@ -1,15 +1,15 @@
 import type { ReactElement } from 'react';
 import type { CSSObject } from 'styled-components';
 import styled from 'styled-components';
-import type { StyledBadgeProps } from './StyledBadge';
+import type { StyledBadgeProps } from './StyledBadge.d';
```

Comment:
Every `.d.ts` file imports now has to explicitly state the `.d` part since StyleBadge in here will point to `.web` or `.native`